src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -61,12 +61,5 @@ def sessions_errored_set(metric_ids, alias=None):
     )
 
 
-def percentage(arg1_snql, arg2_snql, metric_ids, alias=None):
-    return Function(
-        "multiply",
-        [
-            100,
-            Function("minus", [1, Function("divide", [arg1_snql, arg2_snql])]),
-        ],
-        alias,
-    )
+def percentage(arg1_snql, arg2_snql, alias=None):
+    return Function("minus", [1, Function("divide", [arg1_snql, arg2_snql])], alias)
```

Comment:
Should we rename this from percentage to `ratio`? Or actually, since it's `1 - ratio` maybe we should just call it `crash_free_rate`?

Reply:
I'd rather not name it crash_free_rate.. this is meant to be re-used, and I will reuse it later on for users crash free rate 
I could name it ratio but to me its still a percentage, just without the 100 multiplication

Reply:
I could potentially call it rate but I am not sure that is so good because we will have a rate that means something completely different

Reply:
I approved the PR because this is just an internal name so it does not really matter. I do believe that if the function is intended for reuse, a more descriptive name (especially because of the `1 - x`) would be better, though.